cmd/noms/noms.go

```diff
@@ -44,49 +29,70 @@ var actions = []string{
 	"nomming on",
 }
 
-func usageString() string {
-	i := rand.New(rand.NewSource(time.Now().UnixNano())).Intn(len(actions))
-	return fmt.Sprintf(`Noms is a tool for %s Noms data.`, actions[i])
+// CommandHandler - a callback passed to commands
```

Comment:
oops, not vet, golint. I can probably just turn that off, doesn't seem like we follow it.